### DISCUSSION TITLE
AAB missing 64bit .so file of an included ane v33.1.1.698

### DISCUSSION THREAD

**Discussion by teotigraphix** [airsdk/Adobe-Runtime-Support#1424] [Q&A] [answered]
Hey!

I have a personal ane that has Android 32/64, Windows C/C++ libs that are included in my apps. I tried creating an aab using v698 and the .so is copied to the correct place in the `base/lib` directory for the `armeabi-v7a` version but not the  `arm64-v8a`.

Have you seen this at all?

**Comment by ajwfrost** [comment, marked as answer]
Hi

Can I check you packaged the ANE up using 33.1.1.x as well? I'm wondering whether it's possible for you to send us the .ane file so that we can try to reproduce and see what might be going on..

thanks
  Andrew

**Reply by teotigraphix** [reply]
Sure! How should I get it to you?

**Reply by ajwfrost** [reply]
Please use https://transfer.harman.com/requests/P2BO8UkB9DO89EPsQm95C4
thanks

**Reply by teotigraphix** [reply]
Sent.

**Reply by ajwfrost** [reply]
Hi @teotigraphix  - thanks for this, it did expose a bug in the handling of the "libs" subfolders where we were missing some of the different platform variants. A fix is just going into the ADT tool for this...

thanks

**Reply by teotigraphix** [reply]
Right on! Love to help this way of course. :)
